src/app/core/store/navigation/navigation.effects.ts

```diff
@@ -67,7 +67,7 @@ export class NavigationEffects {
     ofType<NavigationAction.RemoveCollectionFromQuery>(NavigationActionType.REMOVE_COLLECTION_TO_QUERY),
     mergeMap(action => this.store$.select(selectQuery).pipe(
       skipWhile(query => !query),
-      first(),
+      take(1),
```

Comment:
Were there any problems with this? Because it should behave the same way.

Reply:
>take(1) emits 1, completes, unsubscribes
>first() emits 1, completes, but doesn't unsubscribe.
according to:
https://stackoverflow.com/questions/42345969/angular-2-using-rxjs-take1-vs-first

Reply:
It seems that description is not very accurate (see the comments there). The difference is better explained [here](https://swalsh.org/blog/rxjs-first-vs-take1/).